### DISCUSSION TITLE
How to deploy docs with versioning

### DISCUSSION THREAD

**Discussion by MartinHugh** [squidfunk/mkdocs-material#6893] [General]
I have an MKDocs site and can use "mike server" to run it locally with versioning.

However what I need to do is to be able to run this under our own web server rather than have to use mike.

When I use "mike deploy" to produce a deployment and then run the site subdirectory on our web server, the pages are served but no versioning is available.

I also tried to produce a single static file (html or pdf) which represents the whole site using  the "print-site" plug-in to do this, and it does produce a single file, but again ... there is no versioning available.

Can anyone help

**Comment by kamilkrzyskow** [comment]
Hello @MartinHugh,
please don't add further discussions if you find a matching one, there are many discussions related to mike:
- https://github.com/squidfunk/mkdocs-material/discussions?discussions_q=mike+deploy

and I've answered you in:

- https://github.com/squidfunk/mkdocs-material/discussions/5124#discussioncomment-8740371

I don't think this discussion adds an unique case, but I'm not sure, so I won't lock the discussion yet.
